experiment_scheduler/master/master.py

```diff
@@ -94,6 +97,25 @@ def get_task_managers() -> List[str]:
             )
         return address
 
+    def _health_check(self, interval=1) -> None:
+        """
+        This thread_running_function periodically checks status and tasks of task_manager
+        If some task managers are abnormal, tasks of the abnormal task manager are retried
+        :param interval: time interval
+        :return: None
+        """
+        while True:
+            self.logger.info("health check start")
+            if self.process_monitor.selected_task_manager != -1:
+                unhealthy_task_manager_list = []
+                for task_manager in self.task_managers_address:
+                    if not self.process_monitor.thread_queue[f"is_{task_manager}_healthy"]:
+                        unhealthy_task_manager_list.append(task_manager)
+                if len(unhealthy_task_manager_list) > 0:
+                    for unhealthy_task_manager in unhealthy_task_manager_list:
+                        self._retry_execute_task(unhealthy_task_manager)
+            time.sleep(interval)
+
```

Comment:
Master check the status-info of task_managers.
If there are abnormal task_managers, then master retry execute task.